src/SprykerEco/Service/ComputopApi/Mapper/ComputopApiMapper.php

```diff
@@ -216,6 +216,6 @@ protected function createUniqueSalt(): string
      */
     protected function getItemDescription(ItemTransfer $itemTransfer): string
     {
-        return sprintf('Name:%s-Sku:%s-Quantity:%s', $itemTransfer->getName(), $itemTransfer->getSku(), $itemTransfer->getQuantity());
+        return sprintf('Name:%s-Sku:%s-Quantity:%s', $itemTransfer->getNameOrFail(), $itemTransfer->getSkuOrFail(), $itemTransfer->getQuantityOrFail());
```

Comment:
if these changes are not required by some CI tool, then revert them to keep backward compatibility
these fields were not required before

Reply:
Fixed